design-system/react/src/components/Input/styles.ts

```diff
@@ -20,8 +20,9 @@ export const input = css({
   overflow: 'hidden',
 
   [`&:focus-within`]: {
-    outline: `2px solid ${OUTLINE_COLOR}`,
-    outlineOffset: '1px',
+    borderColor: '$gray6',
+    // outline: `3px solid ${OUTLINE_COLOR}`,
```

Comment:
Can we remove this comment out, if they are not needed?

Reply:
✅